core/src/test/scala/precog/SbtPrecogBaseSpec.scala

```diff
@@ -160,7 +160,7 @@ class SbtPrecogBaseSpec(params: CommandLine) extends Specification {
     }
 
     "log 'version: revision' for slamx" in {
-      val versions = ManagedVersions(tmpdir.resolve("slamx.json"))
+      val versions = ManagedVersions(tmpdir.resolve("slamx-revision.json"))
```

Comment:
This is also resolvable by making each test entirely independent, rather than sharing `tmpdir.